tensorflow/core/common_runtime/eager/execute.cc

```diff
@@ -800,15 +800,15 @@ Status EagerRemoteExecute(EagerOperation* op, TensorHandle** retvals,
   DataTypeVector output_dtypes;
   TF_RETURN_IF_ERROR(GetOutputDTypes(op, &output_dtypes));
 
-  const size_t num_outputs = static_cast<int>(output_dtypes.size());
-  if (num_outputs != *num_retvals) {
+  const size_t num_outputs = output_dtypes.size();
+  if (static_cast<int>(num_outputs) != *num_retvals) {
```

Comment:
Why not leave the case where it is? And only change type to `int` instead of `size_t`?